### PR TITLE
Add euporie

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
Waiting for https://github.com/conda-forge/pillow-feedstock/pull/93

**Comment by jan-janssen** [comment]
In the CI environment `euporie --help` fails with: 
```
Traceback (most recent call last):
  File "/home/conda/staged-recipes/build_artifacts/euporie_1621062335329/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/bin/euporie", line 10, in <module>
    sys.exit(App.launch())
  File "/home/conda/staged-recipes/build_artifacts/euporie_1621062335329/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib/python3.9/site-packages/euporie/app.py", line 567, in launch
    app = cls()
  File "/home/conda/staged-recipes/build_artifacts/euporie_1621062335329/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib/python3.9/site-packages/euporie/app.py", line 62, in __init__
    super().__init__(
  File "/home/conda/staged-recipes/build_artifacts/euporie_1621062335329/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib/python3.9/site-packages/prompt_toolkit/application/application.py", line 314, in __init__
    self._merged_style = self._create_merged_style(include_default_pygments_style)
  File "/home/conda/staged-recipes/build_artifacts/euporie_1621062335329/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib/python3.9/site-packages/euporie/app.py", line 89, in _create_merged_style
    if self.bg_color:
  File "/home/conda/staged-recipes/build_artifacts/euporie_1621062335329/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib/python3.9/site-packages/euporie/term.py", line 132, in bg_color
    if result := self._query_term(QueryCodes.background_color):
  File "/home/conda/staged-recipes/build_artifacts/euporie_1621062335329/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/lib/python3.9/site-packages/euporie/term.py", line 83, in _query_term
    tattr = termios.tcgetattr(stdin)
termios.error: (25, 'Inappropriate ioctl for device')
```

**Comment by jan-janssen** [comment]
The windows tests fail with:
```
prompt_toolkit.output.win32.NoConsoleScreenBufferError: No Windows console found. Are you running cmd.exe?
```

Still as the package is pure python I would prefer it to be a `noarch` package.

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/euporie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - prompt-toolkit >=3.0.18,<4.0.0
+    - python >=3.6,<4.0
+    - rich >=10.1.0,<11.0.0
+
+test:
+  imports:
+    - euporie
+    - euporie.libs.commonmark_extensions
+  commands:
+    # - pip check
```
Comment (xhochy) [inline recipes/euporie/meta.yaml:39]: Can you add a comment why this us commented?

Comment (jan-janssen) [reply recipes/euporie/meta.yaml]: Done - basically I am waiting for pillow 8.2.0 to be released as conda package https://github.com/conda-forge/pillow-feedstock/pull/93

**Pull request merged**